releases/dev.yml

```diff
@@ -14,12 +14,11 @@ spec:
     metadata:
       labels:
         app: digitalocean-cloud-controller-manager
-      annotations:
-        scheduler.alpha.kubernetes.io/critical-pod: ''
     spec:
       dnsPolicy: Default
       hostNetwork: true
       serviceAccountName: cloud-controller-manager
+      priorityClassName: system-cluster-critical
```

Comment:
I suppose `priorityClassName` is understood by all currently supported Kubernetes releases, i.e., 1.21+?

Reply:
i think this is available since 1.16 : https://github.com/kubernetes/kubernetes/blob/cf04a35e2ab4010c25268106f403ddb46df0b497/CHANGELOG/CHANGELOG-1.16.md

Reply:
```
the scheduler.alpha.kubernetes.io/critical-pod annotation is removed. Pod priority (spec.priorityClassName) should be used instead to mark pods as critical. ([#80342](https://github.com/kubernetes/kubernetes/pull/80342), [@draveness](https://github.com/draveness))
```

Reply:
Thanks for confirming 👍